Libs/MRML/Core/vtkMRMLSubjectHierarchyNode.cxx

```diff
@@ -3654,3 +3654,13 @@ vtkMRMLSubjectHierarchyNode* vtkMRMLSubjectHierarchyNode::GetSubjectHierarchyNod
 
   return scene->GetSubjectHierarchyNode();
 }
+
+//----------------------------------------------------------------------------
+void vtkMRMLSubjectHierarchyNode::ShowItemsInView(vtkIdList* itemIDs, vtkMRMLAbstractViewNode* viewNode)
+{
+  SubjectHierarchyItemsShowInViewRequestedEventData eventData;
+  eventData.itemIDsToShow = itemIDs;
+  eventData.viewNode = viewNode;
+  this->InvokeEvent(SubjectHierarchyItemsShowInViewRequestedEvent, &eventData);
```

Comment:
Is this struct available to an observer in python using the decorator?  If not we could use a vtkObject subclass instead.

Reply:
The SubjectHierarchyItemsShowInViewRequestedEvent (and the associated event data struct) is for internal use only, to allow qMRML widgets to call qSlicer functions (qMRML classes cannot directly depend on qSlicer classes).

vtkMRMLSubjectHierarchyNode::ShowItemsInView or qSlicerSubjectHierarchyPluginHandler::showItemsInView method can be used to request view of subject hierarchy items in a view.

I'll add comments to clarify this.